### DISCUSSION TITLE
Concat from two different databases

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#15653] [Q&A] [answered]
Is it possible to do a single cross database query. We have two databases one containing archive data and one containing live production data.

Can a query be composed that executes a query on both and returns a single result set.

At the moment we open a session on each DB, do separate queries and then combine the results manually.

**Comment by DejanMilicic** [comment, marked as answer]
You cannot do it directly.
You would need to first join data together in one database and then perform such a query.

Currently, we are working on the Data Archival feature https://issues.hibernatingrhinos.com/issue/RavenDB-19501/Data-archival-support 
which should support this scenario. You would set up Data Archival process that would store Orders into Orders_Archived collection, and then you would write multi-map index that would process both current and archived Orders.

In the meantime, you could also use RavernDB ETL to achieve similar effect.
